File: src/main/java/com/lmp/notification/dto/InAppNotificationDto.java
package com.lmp.notification.dto;

/**
 * DTO pour les notifications in-app retournées au frontend.
 */
public record InAppNotificationDto(
        String id,
        String type,
        String message,
        String orderId,
        String serviceName,
        Double amount,
        boolean read,
        String timestamp
) {}

File: backend-dotnet/src/Lmp.Application/Notification/InAppNotificationDto.cs
namespace Lmp.Application.Notification;

/// <summary>In-app notification DTO. Faithful port of <c>com.lmp.notification.dto.InAppNotificationDto</c>.</summary>
public sealed record InAppNotificationDto(
    string Id,
    string Type,
    string Message,
    string? OrderId,
    string? ServiceName,
    double? Amount,
    bool Read,
    string Timestamp);
